apps/editor/src/__test__/unit/convertor.spec.ts

```diff
@@ -1098,4 +1098,34 @@ describe('Convertor', () => {
 
     expect(result).toBe(`<strong>"test"</strong>a`);
   });
+
+  it('should convert empty line which is between lists of wysiwig to <br>', () => {
```

Comment:
```suggestion
  it('should convert empty line between lists of wysiwig to <br>', () => {
```

참고: https://www.grammarly.com/blog/in-between-or-inbetween/